src/stylesheets/mixins/_auth-mixins.scss

```diff
@@ -0,0 +1,9 @@
+@mixin auth-container() {
+  align-items: center;
```

Comment:
Do we need this mixin somewhere, if it will be used in just one place, it would be better if we can remove it and move the style to that place directly.

Reply:
Yes, I plan to use it in `reset-password` and `forgot-password` screen as well. That's why I made it a mixin.